bidwire/tests/test_commbuys_scraper.py

```diff
@@ -1,7 +1,7 @@
 import os
 import pytest
 
-from scraper import scrape_results_page
+from commbuys_scraper import CommBuysScraper
```

Comment:
Please add a similar basic test for the cityofboston scraper.

(In general, as we add more code I'd like us to get better at adding tests with it.)